builtin/io_osh.py

```diff
@@ -10,6 +10,7 @@
 from frontend import flag_spec
 from frontend import lexer
 from frontend import match
+from frontend.typed_args import DoesNotAccept
```

Comment:
Our C++ translation (mycpp) usually requires `from frontend import typed_args`

there are a couple exceptions but that's the general rule